src/components/Nav/Nav.js

```diff
@@ -1,7 +1,55 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { FaSearch } from 'react-icons/fa';
+import * as S from './Nav.style';
 
 const Nav = () => {
-  return <div>Nav</div>;
+  const navigate = useNavigate();
+  const login_token = localStorage.getItem('login-token');
+
+  const logout = () => {
+    localStorage.removeItem('login-token');
+    navigate('/');
+  };
+  return (
+    <S.NavContainer>
+      <S.NavWrapper>
+        <S.Home>
+          <S.StyledHomeLink to="/">Wevre</S.StyledHomeLink>
+        </S.Home>
+        <S.MenuList>
+          {MENU_LIST.map(li => {
+            if (li.id === 2) {
+              return login_token ? (
+                <S.StyledMenuLink key={li.id} onClick={logout}>
+                  {li.text[1]}
+                </S.StyledMenuLink>
+              ) : (
+                <S.StyledLink key={li.id} to={li.link}>
+                  {li.text[0]}
+                </S.StyledLink>
+              );
+            } else {
+              return (
+                <S.StyledLink key={li.id} to={li.link}>
+                  {li.text}
+                </S.StyledLink>
+              );
+            }
+          })}
```

Comment:
```suggestion
          {MENU_LIST.map(({id,text,link}) => {
            if (id === 2) {
              return login_token ? (
                <S.StyledMenuLink key={id} onClick={logout}>
                  {text[1]}
                </S.StyledMenuLink>
              ) : (
                <S.StyledLink key={id} to={link}>
                  {text[0]}
                </S.StyledLink>
              );
            } else {
              return (
                <S.StyledLink key={id} to={link}>
                  {text}
                </S.StyledLink>
              );
            }
          })}
```
구조분해할당을 적절히 써주시면, 조금 더 간결해 집니다.